include/random_walks/gaussian_hamiltonian_monte_carlo_exact_walk.hpp

```diff
@@ -10,10 +10,17 @@
 
 #include "sampling/sphere.hpp"
 
+#include <iostream>
 
+typedef long double LNT;
+typedef double NT;
 
 // Exact HMC for sampling from the spherical Gaussian distribution
 
+
+//LNT _Len;
+NT _Len;
```

Comment:
please do not use global variables, it is error-prone and we avoid it in the whole library.